.github/workflows/update_modules.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         run: python ./_repo_generator.py
       - name: Commit packages
         run: |
-          git add *.zip
```

Comment:
If this doesn't work, the [documentation](https://git-scm.com/docs/git-add#_examples) suggests `git add \*.zip`.